src/Core/Cache/CacheFactory.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace SilverStripe\Core\Cache;
+
+use Psr\SimpleCache\CacheInterface;
+
+interface CacheFactory
```

Comment:
We don't need a `CacheFactory`. Just use `SilverStripe\Core\Injector\Factory` directly.